welcome/learn_more_about_openshift.adoc

```diff
@@ -180,6 +188,7 @@ a|* xref:../networking/networking_operators/cluster-network-operator.adoc#nw-clu
 [discrete]
 ==== Changing cluster components
```

Comment:
This section the first row. 2 of the links are the same and go to updating using web console. remove one.